baremetal/polarfire-soc-bare-metal-library/src/platform/drivers/mss_qspi/mss_qspi.c

```diff
@@ -1,5 +1,5 @@
 /***************************************************************************//**
- * Copyright 2019-2020 Microchip FPGA Embedded Systems Solutions.
+ * Copyright 2019-2021 Microchip FPGA Embedded Systems Solutions.

```

Comment:
*question*: I believe there were a few files that had the copyright changed, but I only highlighted one. Would you mind taking a quick look through please?

Reply:
Changed the copyright in 4 files, added missing new line at end of 1 file